packages/provider_devtools_extension/lib/src/provider_screen.dart

```diff
@@ -42,71 +46,97 @@ class ProviderScreenBody extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    final splitAxis = Split.axisFor(context, 0.85);
+    final splitAxis = SplitPane.axisFor(context, 0.85);
 
     // A provider will automatically be selected as soon as one is detected
     final selectedProviderId = ref.watch(selectedProviderIdProvider);
     final detailsTitleText = selectedProviderId != null
         ? ref.watch(_selectedProviderNode)?.type ?? ''
         : '[No provider selected]';
 
-    ref.listen<bool>(_hasErrorProvider, (_, hasError) {
-      if (hasError) showProviderErrorBanner();
+    ref.listen<AsyncError?>(_hasErrorProvider, (_, hasError) {
+      if (hasError != null) {
```

Comment:
As I said above, the provider name makes no sense to me anymore. So the value needs to be renamed too.

We don't have an "hasError" value but the raw error itself.